Source/WebKit/Shared/API/Cocoa/_WKHitTestResult.h

```diff
@@ -45,6 +45,7 @@ WK_CLASS_AVAILABLE(macos(10.12), ios(16.0))
 
 @property (nonatomic, readonly, copy) NSURL *absolutePDFURL;
 @property (nonatomic, readonly, copy) NSURL *absoluteLinkURL;
+@property (nonatomic, readonly, copy) NSString *linkDataMIMEType WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA));
```

Comment:
Ideally, the name of the property would also indicate that it only applies for local data.

Maybe `linkLocalDataMIMEType`?